HippUnfoldHCP/PostHippUnfoldHCP.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
+set -eu
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+  # pipedirguessed=1
+   #fix this if the script is more than one level below HCPPIPEDIR
+   export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+
+opts_SetScriptDescription "Make some BIDS structures and run HippUnfold"
+
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects"
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddOptional '--hippunfold-dir' 'PhysicalHippUnfoldDIR' 'path' "location of HippUnfold outputs"
+opts_AddOptional '--atlas-hippunfold-dir' 'AtlasHippUnfoldDIR' 'path' "location of Atlas HippUnfold outputs"
+
+opts_ParseArguments "$@"
+
+if ((pipedirguessed))
+then
+    log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+opts_ShowValues
+
+log_Msg "Starting PostHippUnfold pipeline for subject: $Subject"
+
+T1wFolder="$StudyFolder/$Subject/T1w"
+AtlasFolder="$StudyFolder/$Subject/MNINonLinear"
+
+if [ -z ${PhysicalHippUnfoldDIR} ] ; then
+  PhysicalHippUnfoldDIR="${T1wFolder}/HippUnfold"
+fi 
+
+if [ -z ${AtlasHippUnfoldDIR} ] ; then
+  AtlasHippUnfoldDIR="${AtlasFolder}/HippUnfold"
+fi 
+
+function PostHippUnfold {
+HippUnfoldFolderOut=${1}
+PhysicalHippUnfoldFolder=${2}
+AtlasHippUnfoldFolder=${3}
+T1wFolder=${4}
+AtlasFolder=${5}
+Subject=${6}
+Modality=${7}
+Mesh=${8}
+
+if [ ${Modality} = "T1wT2w" ] ; then
+  Modality="T1w"
+  Flag="On"
+else
+  Flag="Off"
+fi
```

Comment:
You could also consider this pattern:
```suggestion
EffectiveModality="$Modality"
if [ ${Modality} = "T1wT2w" ] ; then
  EffectiveModality="T1w"
fi
```

Reply:
Looks like `local` works to me:
```
$ function foo() { local bar=broken; }
tim@blackbox:~$ bar=fixed
tim@blackbox:~$ foo
tim@blackbox:~$ echo $bar
fixed
```

Reply:
But in general, these problems don't exist if you just use a new variable name rather than destroying a value you need again later.

Reply:
To be clear, the very first use of each variable in the function needs the local keyword to do this, but later uses don't:

```
$ function foo() { local bar=broken; bar=morebroken; }
$ bar=fixed
$ foo
$ echo $bar
fixed
```
If it isn't on the first one, you get a problem:
```
$ function foobar() { bar=oops; local bar=okay; }
$ bar=fixed
$ foobar
$ echo $bar
oops
```